host_addr: Vicon
tf_ref_frame: /odom
subject_list: [txt01]
